add a new notebook to show ag time series with sandbox data/stats api

matthew-ballard opened the pull request:
republished an old notebook to show how to use PlanetScope with Statistical API - now uses sandbox data

adripollack commented:
Hey Matt! Here are some notes from a first pass, feel free to take or leave though as the intended workflow demonstration is good and clear--

Markdown/comments:
- on the note `# Specify a resolution - in the units of the provided areas of interest, in this case EPSG:4326` --it could be useful to mention that this is the resolution of the imagery itself.  I've never quite known why this argument was required, but deriving it can be a bit of a pain in case you have any suggestions for folks there if they don't know these settings off the top of their head
- which lines/acts consume PUs?  Can the command(s) be labeled?  I'd imagine some users would be interested in that callout before hitting the line that states PU consumption

Cluster plots:
- A note on what might explain different clusters can be useful, i.e. differences in crop types, fallowed vs. planted fields
- Can dates be shifted at all or is the sandbox limited to 2022-05 to 2023-05?  I believe you're capturing two different crop types over this range and shifting to a single growing season (2022-03 to 2022-09) could give better clusters and AUC, along with timeseries that have nice greenup features that come before the season's phenology curve rather than after
- the AUC exercise mentions constraining the data to a specific time range, if time ranges can't shift to a single growing season I'd think AUC can be constrained to a single season

matthew-ballard commented:
thank you!! 

- added some details to explain the spatial resolution is for the imagery and that it can be adjusted, e.g. to do analysis on downsampled data
- added a callout for the step that consumes PUs, great point and we should do something similar by default.
<div class="alert alert-block alert-info">
<b>Processing Units:</b> The following code block will consume processing units.
</div>

Cluster plots
- What different clusters represent - I don't have data on what the fields actually represent, unfortunately
- Unfortunately the sandbox data is two halves of a year and not a whole calendar year/growing season.  I will mark this down as a suggestion to improve our sandbox data for the next update that we do.
- I updated these to only go from Jan->mid year.

adripollack reviewed (approved):
looks good to me!
